Src/IronPython/Compiler/Parser.cs

```diff
@@ -1053,13 +1054,19 @@ public class Parser : IDisposable { // TODO: remove IDisposable
             // Save private prefix
             string savedPrefix = SetPrivatePrefix(name);
 
+            var ret = new ClassDefinition(name, bases.ToArray(), keywords.ToArray(), metaclass: metaclass);
```

Comment:
Maybe I'm missing something in this file, but other than changing the code to match `FunctionDefinition`, is there anything else going on?